src/get-activity.ts

```diff
@@ -18,7 +18,7 @@ import {
   getPullRequestReviews,
 } from "./start";
 
-enum CommentType {
+export enum CommentType {
   REVIEW = 0b1,
   ISSUE = 0b10,
   ASSIGNEE = 0b100,
```

Comment:
Tell me you're a C developer without telling me you're a C developer. Although this is interesting, perhaps for readability it makes sense to use less efficient strings instead.

You would probably enjoy Solidity as these optimizations are valuable for lowering gas fees!